src/controller_initializer.coffee

```diff
@@ -32,5 +32,13 @@ class ControllerInitializer
       controllerName = camelCase Controller.name.replace /Controller$/, ''
       @controllers[controllerName] = new Controller @dependencies
 
+  # Finds any middleware defined for the controller action
+  middlewareFor: ({controllerName, actionName}) ->
+    controller = @controllers[camelCase controllerName]
+    return [] unless controller?
+    return [] unless typeof controller[actionName] is 'function'
+    return [] unless controller.middleware?[actionName]?
+    [].concat controller.middleware[actionName]
```

Comment:
Awesome! So this means we can pass in an array of middleware instead of just a single one if needed?